frontend/javascripts/dashboard/advanced_dataset/dataset_action_view.js

```diff
@@ -135,18 +137,32 @@ class DatasetActionView extends React.PureComponent<Props, State> {
       </Dropdown>
     );
 
+    const reloadOption = !dataset.isForeign ? (
+      <a
+        href="#"
+        onClick={() => this.clearCache(dataset)}
+        title="Reload Dataset"
+        style={disabledWhenReloadingStyle}
+      >
+        {isReloading ? <Icon type="loading" /> : <Icon type="retweet" />}
+        Reload
+      </a>
+    ) : null;
+
     return (
       <div>
         {dataset.isEditable && dataset.dataSource.dataLayers == null ? (
-          <div>
-            <Link
-              to={`/datasets/${dataset.owningOrganization}/${dataset.name}/import`}
-              className="import-dataset"
-            >
-              <Icon type="plus-circle-o" />
-              Import
-            </Link>
-
+          <div className="dataset-actions">
+            <React.Fragment>
```

Comment:
The `<React.Fragment>` should not be needed as far as I can see (since there is the wrapping div).